apps/connect/src/App.tsx

```diff
@@ -28,15 +33,23 @@ export default function Root() {
       bridgeDefaults: {
         ...(sourceChain && { fromNetwork: sourceChain as ChainName }),
         ...(targetChain && { toNetwork: targetChain as ChainName }),
+        ...(token && { token: token as string }),
```

Comment:
I think would be good to add a validation over this token to do not end with bad behaviour

Reply:
Ready!